tests/scaling-pipelines/scenario/git-resolver/task.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: echo
+  namespace: benchmark
```

Comment:
- This script doesn't seem to be applied in **setup.sh**. For the cluster-resolver, we created a new method [oc_apply](https://github.com/openshift-pipelines/performance/blob/0100ce43f7d9ca082b64f30fb38f1e094c092c23/tests/scaling-pipelines/scenario/cluster-resolver/setup.sh#L21C1-L21C28) that manually applies the file during the setup.
- For running the tests on multi-namespace, we need to set this to individually _benchmark1/2/3..._ namespaces.